nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/pom.xml

```diff
@@ -230,6 +230,12 @@ language governing permissions and limitations under the License. -->
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.mockftpserver</groupId>
```

Comment:
@trixpan is correct on the LICENSE/NOTICE for binary dependencies like this.  If we're pulling in source even for test purposes though then we have to account for it.